Presentation/PresentationStyle.swift

```diff
@@ -237,6 +237,7 @@ private extension PresentationStyle {
                 case .left(let view):
                     popover?.sourceView = view
                     popover?.sourceRect = view.bounds
+                    popover?.backgroundColor = .white
```

Comment:
Shouldn't we add this as a defaulted parameter instead that defaults to nil (standard behavior)? 
Shouldn't backgroundColor be updated no mater if we present from a view or item?
Another alternative would be to add a PopoverStyle with e.g. backgroundColor, permittedDirections, layoutMargins and other styling parameters.

Reply:
You are right, I added the popover background colour to white by default no matter what we present. Regarding styling we could do something like this:
```
let popoverStyle = PresentationStyle.popover(from: button, permittedDirections: .up) { vc in
                        // custom styling on vc.popoverPresentationController goes here
}
```
So, although introducing PopoverStyle would be nice, there is already a way to customise it.